src/app/services/age-verification/age-verification.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AgeVerificationService {
+  readonly LEGAL_AGE = 18;
+
+  ageVerification(age: number, firstName: string, lastname: string): string {
```

Comment:
`ageVerification` is the name of the service, not of the method. What if you change to `performAgeVerification`, or another good name?